uswds/uswds-site
# USWDS-Site: Transfer showcase content from USWDS repo and repair HTMLProofer error

## Summary
1. Transferred "Showcase" content from the `uswds` repo into `uswds-site` 
    - Related to issue https://github.com/uswds/uswds/issues/3324
1. Moved showcase content to `showcase.md` instead of `showcase-all.md`
1. Removed `showcase-all.md` and related references.
1. Removed references to `WHO_IS_USING_USWDS.md`
1. Removed the link to the  NASA Web Style Guide because it was returning a 404 and HTMLProofer error 
    - Related to #1990
    
The `uswds` repo update that removes the `WHO_IS_USING_USWDS.md` file can be found in PR https://github.com/uswds/uswds/pull/5120. 
:warning: This PR should be merged before merging https://github.com/uswds/uswds/pull/5120.

## Related issue
Closes #1990
Related to https://github.com/uswds/uswds/issues/3324

## Preview link
- [Showcase](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-move-showcase-info/documentation/showcase)
- [What's New](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-move-showcase-info/about/whats-new/)
- [Next](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-move-showcase-info/next/get-involved/)

## Solution
Issue #1990 revealed a 404 error with one of the links listed in `WHO_IS_USING_USWDS.md` in the `uswds` repo. 

Instead of repairing this issue in the `uswds` repo, this PR moves the content from `WHO_IS_USING_USWDS.md` directly into the showcase page and repairs the issue here. 

As described in https://github.com/uswds/uswds/issues/3324:
> There's no intrinsic relationship between the showcase data and USWDS source code, principles, guidance, etc. We should move this data to uswds-site to make it easier to update.

This PR also removes the unnecessary `showcase-all` subpage and updates all references to both `showcase/all` and `WHO_IS_USING_USWDS.md`.

## Testing and review
To review, confirm that:
1. All content appears as expected on the Showcase page.
1. Confirm that `npm run proof` returns no errors
1. Confirm that all references to `showcase/all` and `WHO_IS_USING_USWDS.md` have been appropriately updated.
1. Confirm that no suitable replacement exists for the NASA Web Style Guide link.
